lib/partitioninfo.coffee

```diff
@@ -43,18 +45,24 @@ getLogicalPartitions = (disk, offset, extendedPartitionOffset = offset, limit =
 					result
 		result
 
-getPartitions = (disk, offset, getLogical = true) ->
+getPartitions = (disk, includeExtended, offset, getLogical = true) ->
```

Comment:
@jviotti updated, the prototype is now (disk, options).
options is { includeExtended, offset, getLogical }